pkg/binpack/distribute_evenly_test.go

```diff
@@ -30,10 +31,23 @@ func createResources(cpu, memory, nvidiaGPU int64) *resources.Resources {
 		NvidiaGPU: *resource.NewQuantity(nvidiaGPU, resource.BinarySI),
 	}
 }
+
 func createSchedulingMetadata(cpu, memory, nvidiaGPU int64, zoneLabel string) *resources.NodeSchedulingMetadata {
 	return &resources.NodeSchedulingMetadata{
-		AvailableResources: createResources(cpu, memory, nvidiaGPU),
-		ZoneLabel:          zoneLabel,
+		AvailableResources:   createResources(cpu, memory, nvidiaGPU),
+		SchedulableResources: createResources(math.MaxInt64, math.MaxInt64, math.MaxInt64),
+		ZoneLabel:            zoneLabel,
+	}
+}
+
+func createSchedulingMetadataWithTotals(
+	availableCPU, cpuTotal, availableMemory, memoryTotal, availableNvidiaGPU, nvidiaGPUTotal int64,
+	zoneLabel string) *resources.NodeSchedulingMetadata {
+
+	return &resources.NodeSchedulingMetadata{
+		AvailableResources:   createResources(availableCPU, availableMemory, availableNvidiaGPU),
+		SchedulableResources: createResources(cpuTotal, memoryTotal, nvidiaGPUTotal),
+		ZoneLabel:            zoneLabel,
 	}
```

Comment:
Lets move this closer to where it's actually used

Reply:
Ack. Moved this to resources where it should belong.